packages/cursorless-everywhere-talon-core/src/ide/setSelections.ts

```diff
@@ -0,0 +1,17 @@
+import type { Selection, TextDocument } from "@cursorless/common";
+import type { Talon } from "../types/talon.types";
+
+export function setSelections(
+  talon: Talon,
+  document: TextDocument,
+  selections: Selection[],
+): Promise<void> {
+  const selectionOffsets = selections.map((selection) => ({
+    anchor: document.offsetAt(selection.anchor),
+    active: document.offsetAt(selection.active),
+  }));
+
+  talon.actions.user.cursorless_everywhere_set_selections(selectionOffsets);
+
+  return Promise.resolve();
```

Comment:
i've seen you doing this in a couple places in this PR. Is this really better than just adding an `async` keyword to the function?

Reply:
I don't know? Generally the linter will be angry if you have an async function without an await in it so I kinda Learned not to use that, but we might not be using that rule in Cursorless so here it's probably okay.

Reply:
Yeah I've never seen that error

Reply:
I think it is because we haven't enabled the full type checking.